src/main/resources/view/TaskListCard.fxml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+
+<?import javafx.scene.control.CheckBox?>
+<?import javafx.scene.image.ImageView?>
+<HBox styleClass="tasks_container" id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/20.0.1" xmlns:fx="http://javafx.com/fxml/1">
+    <GridPane HBox.hgrow="ALWAYS">
+        <columnConstraints>
+            <ColumnConstraints hgrow="SOMETIMES" minWidth="10"/> <!-- Adjusted width -->
+            <ColumnConstraints hgrow="SOMETIMES" minWidth="10" halignment="RIGHT" /> <!-- Right-align column -->
+        </columnConstraints>
+        <VBox alignment="CENTER_LEFT" minHeight="40" GridPane.columnIndex="0">
+            <HBox alignment="CENTER" spacing="5">
+                <Label fx:id="id" styleClass="task_label" />
+                <CheckBox fx:id="isDone" disable="true" styleClass="task_label"/>
+
+                <Label fx:id="name" styleClass="task_label" text="\$first"/>
+            </HBox>
+        </VBox>
+        <VBox alignment="CENTER_RIGHT" minHeight="40" GridPane.columnIndex="1"> <!-- Right-align VBox -->
+            <HBox alignment="CENTER" spacing="5">
+                <ImageView fx:id="employeeIcon" fitWidth="15" fitHeight="15" />
+                <Label fx:id="employee" styleClass="task_label" text="\$employee" />
+                <ImageView fx:id="deadlineIcon" fitWidth="15" fitHeight="15" />
+                <Label fx:id="deadline" text="\$deadline" />
+            </HBox>
+        </VBox>
+    </GridPane>
+</HBox>
```

Comment:
![image](https://github.com/AY2324S1-CS2103T-T08-3/tp/assets/63438414/42d7269b-6049-44f7-80cc-6841a17536cc)

It seems that having a longer name (or maybe even other fields) can make the task unaligned with other tasks.